cvat/apps/engine/views.py

```diff
@@ -189,6 +189,12 @@
 _DATA_UPDATED_DATE_HEADER_NAME = 'X-Updated-Date'
 _RETRY_AFTER_TIMEOUT = 10
 
+ICON_FILE = "assets/logo.svg"
+
+def get_logo() -> str:
+    with open(osp.join(osp.dirname(__file__), ICON_FILE)) as f:
+        return f.read()
```

Comment:
I will propose to support .png files also
Thus it may be encoded to base64 on the server and displayed as is on client

Reply:
Implemented solution with static file, so different image formats are supported now